module/cmd/gravity/cmd/genaccounts.go

```diff
@@ -45,7 +45,10 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
 
-			homeFlag, _ := cmd.Flags().GetString(flags.FlagHome)
+			homeFlag, errHomeFlag := cmd.Flags().GetString(flags.FlagHome)
+			if errHomeFlag != nil {
+				fmt.Printf("Homeflag has an error %v", errHomeFlag)
```

Comment:
I think this should be a panic, thoughts @jkilpatr ?

Reply:
I agree in principle, we should try running the command without the flag set and make sure that doesn't produce an error.